packages/vanilla/src/sass/components/banner/_usa.scss

```diff
@@ -92,4 +94,12 @@
 .banner-active {
   display: flex;
   flex-wrap: wrap;
+}
+
+// 433px is the inline size where the text wraps
+@container usa-banner (min-width: 433px) {
+  .cbp-banner__title {
+    max-height: var(--cbp-space-8x);
```

Comment:
this is actually problematic as there are some viewport sizes where there is no space before the expanded content due to this max height being less than the content. Seems to work perfectly fine without it.